docs/_docs/Edison/Introduction/0.6-News.md

```diff
@@ -5,12 +5,7 @@ sidebar: edison
 product: Edison
 ---
 ## What's new for this release
- 1. Linux make 5.10.63 LTS the default kernel, upgrade U-Boot to v2021.10
-    We also provide v5.14.0 and v5.10.59-rt52 (PREEMPT_RT kernel) for development purposes.
- 2. Update to Yocto Gatesgarth.
- 3. Add ppp and openvpn support, see [Networking](4.2-networking.html#openvpn)
- 4. Fix Bitbake to allow signed feeds of DEB packages, see [Setting up for signed DEB repositories](5.0-Creating-a-deb-repository.html#setting-up-for-signed-deb-repositories)
- 5. Create recipe for `zerotier`
- 6. Restore SD card access in U-Boot and Linux
- 7. Finally fix HSU for communication speeds up to 3.5Mb/s, see [HSU](4.4-hsu-i2c-spi.html#hsu)
- 8. Update Debian snapshot to bullseye, see [Debian](2.6-Building-Debian)
+ 1. Linux make 5.15.25 LTS the default kernel, upgrade U-Boot to v2021.10
+    We also provide v5.16.0 (testing) and v5.15.25-rt33 (LTS PREEMPT_RT kernel) for development purposes.
+ 2. Update to Yocto Hardknott.
+ 3. Move `xfstk` recipe to meta-intel-edison-distro to make it visible on xfstk: move to meta-intel-edison-distro
```

Comment:
This is obviously a type, should say to make it visible on [https://layers.openembedded.org](https://layers.openembedded.org/layerindex/branch/hardknott/recipes/)